.gitignore

```diff
@@ -25,12 +23,11 @@ bin/
 gen/
 out/
 
-# Gradle files
-.gradle/
-build/
+# Gradle build files
+matisse/build
+sample/build
+
 
-# Local configuration file (sdk path, etc)
-local.properties
```

Comment:
local properties should be local, right? Say I saved some key and secret in this file, it shouldn't be exposed.